cmd/plugins/memtierd/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG GO_VERSION=1.21
+
+FROM golang:${GO_VERSION}-bullseye as builder
+
+WORKDIR /go/builder
+
+RUN GOBIN=/bin go install github.com/intel/memtierd/cmd/memtierd@latest
+
+# Fetch go dependencies in a separate layer for caching
+COPY go.mod go.sum ./
+COPY pkg/topology/ pkg/topology/
+RUN go mod download
+
+# Build nri-memtierd
+COPY . .
+
+RUN make clean
+RUN make PLUGINS=nri-memtierd build-plugins-static
+
+# FROM gcr.io/distroless/static
+# ENV PATH=/bin
+# TODO: DEBUG why exec.Command("memtierd", "-c", ...).Start() fails on distroless:
+# execve("/bin/memtierd", ["/bin/memtierd", "-c", "", "-config", "/host/tmp/memtierd/default/test-"...], 0xc000146000 /* 38 vars */) = -1 ENOENT (No such file or directory)
```

Comment:
Just a note that this is probably due to some required dll missing from distroless/static

Reply:
Fantastic! I added `go install` build parameters to build a static version of memtierd. Now we can use distroless/static just fine. Thanks @marquiz!